trie/patriciaMerkleTrie_test.go

```diff
@@ -1708,3 +1709,32 @@ func BenchmarkPatriciaMerkleTrie_RootHashAfterChanging30000NodesInBatchesOf200(b
 		}
 	}
 }
+
+func TestTrieUpdateTimer(t *testing.T) {
```

Comment:
This unit test:
 - is skipped
 - has no assertions
 - does a print line

Instead, let's get the output of this test, or of the existing benchmarks - before & after the fix, and put the output (textual) in the PR description. Then, we can drop this test.

Reply:
yes, we can remove the test after we have the results.

Reply:
Fixed (transformed into a benchmark test).